### ISSUE TITLE
Data Laboratory does not display

### ISSUE THREAD

**Issue by danmeek** [gephi/gephi#1690]
After importing Edges from a .csv file, the Data Laboratory does not appear, leaving a dark gray field.

## Steps to Reproduce

1. Create new gephi project
2. Import .csv file as Edges with 'create missing nodes' selected
3. Wait... and no Data Laboratory opens
4. Close gephi and reopen -- same problem

## Context

I'm trying to recreate another bug, where gephi drops edges apparently at random.

## Your Environment

Product Version: Gephi 0.9.1 201510222201
Java: 1.8.0_74; Java HotSpot(TM) 64-Bit Server VM 25.74-b02
Runtime: Java(TM) SE Runtime Environment 1.8.0_74-b02
System: Mac OS X version 10.11.5 running on x86_64; UTF-8; en_US (gephi)

**Comment by eduramiba** [comment]
Hi,
Can you share an screenshot of what you see? It's possible that you just closed the data laboratory window.

**Comment by danmeek** [comment]
Hi Eduardo,

Thanks for following this up. I think I've worked out what's happening.
Normally in Gephi I'm used to the import from spreadsheet process
finalising, and then Gephi switches to the Data Laboratory and shows the
imported data in a data table sub-window.  What's happening is that the
data table sub-window is not opening after import. If I go to the Window
menu and select Data Table, it appears and then we can work with it.  My
colleague and I have encountered this problem and we thought Gephi had
crashed -- because nothing happens -- so we were closing and restarting
Gephi.

Cheers,
Daniel

**Comment by eduramiba** [comment]
Hi Daniel,

After import, Gephi never changes the view to Data laboratory. So, if you do the import from Overview, you will need to open the Data laboratory group manually.

**Comment by danmeek** [comment]
Hi Eduardo,

Ok - but the confusion has arisen because there's no data table visible
when I click on the button for the data laboratory. Along with my
colleagues, the expected design pattern was import spreadsheet -> go to
datalab -> table appears. As you know, when nothing happened, we thought
Gephi had crashed altogether. (As some analysis programs do - eg SPSS.)

Perhaps this could be feedback for a change in the program behavior after
import?

Many thanks again for your time in responding.

Regards from Australia,
Daniel

**Comment by eduramiba** [comment]
Oh, then the table should be visible, it's possible that you closed it by accident. In that case it's enough to open it again with Window menu.
You can also use the `Window - Reset windows` option to make Gephi look as default.

**Comment by danmeek** [comment]
No... I haven't closed it by accident. I've tried this and it occurs
repeatedly. I'm using Gephi for Mac OS X. After import, no data table
appears in the datalab unless you manually choose to open a data table
window.

Thanks
Daniel

**Comment by alexheeb1** [comment]
I'm having the same problem danmeek had. The data laboratory is blank, even though you can see it on the graph in the overview view. When you right click on a node, the option to view it in the data laboratory is greyed out/can't be clicked.

**Comment by danmeek** [comment]
The workaround we're using is to go to the Window menu and choose Data
Table, then it appears in the Data Laboratory. Does that fix the problem of
the option being greyed out?

**Comment by GCharalampidis** [comment]
The workaround worked really well. Thanks.

**Comment by eschultze** [comment]
Just to mention that this is still happening in Gephi 0.9.2 in macOS Mojave

**Comment by rgriffogoes** [comment]
I can't be really sure but I also faced this "oh no, the Data Laboratory is gone" and got here. Going to window -> data table displays it back and I also don't think I closed it by accident. 0.9.2 here.

My guess is that as I'm constantly re-opening/re-importing a .graphml file (that my app is creating), and one of this imports gave me errors (same Node property with conflicting datatypes -- sometimes integer sometimes string) during import, and once I imported with these critical errors I _think_ the Data Lab was closed.

Another guess is that there was an NullPointerException logged (bottom right), but I only noticed when I decided to check why DataLab was gone (can't really tell when the excp was there).

Will keep my eyes opened to see if I can reproduce this "error" (I guess the DataLab "data table" is not supposed to get closed automatically, right?)

**Comment by ksanz1** [comment]
Hi
I have the same problem. Data laboratory is empty. I´ve tried the diferente solutions you posted here... and nothing changes

**Comment by tautme** [comment]
Hello,

I am having the same problem.

1. Noticed that betweenness centrality was not populating on data table after running, Closeness and Eccentricity did. 
2. Then the entire table would not display in Data Laboritory.
3. Then the top bar with "Nodes, Edges, Config....." also disappeared.

Restarting entire computer didn't seem to help.

I was wondering if there was some kind of underlying reset available?
Also, Do the .giphi_temp files have something to do with this?

Regards, 
Adam

Product Version: Gephi 0.9.2 201709241107
Java: 1.8.0_291; Java HotSpot(TM) 64-Bit Server VM 25.291-b10
Runtime: Java(TM) SE Runtime Environment 1.8.0_291-b10
System: Windows 10 version 10.0 running on amd64; Cp1252; en_US (gephi)

**Comment by mariaubltrn** [comment]
Good day. I also faced this problem and the unfortunate part is there is no Menu bar that is displayed at all.
I'm using the Gephi 0.10.1 for Mac Silicon M2. Need help.

**Comment by furkanayik** [comment]
Version 0.10.1 using on Mac Silicon M1. Data Laboratory is completely blank :/

**Comment by acpn** [comment]
Same issue here, Mac M1 gephi Version 0.10.1

**Comment by IArnaut2** [comment]
I have the same problem too. I drew nodes in a new workspace, but the Data Laboratory does not show.

New workspace with nodes:

<img width="1920" height="1020" alt="Image" src="https://github.com/user-attachments/assets/dca2e37f-48e5-4b12-8b21-b2630e278608" />

Data Laboratory:

<img width="1920" height="1020" alt="Image" src="https://github.com/user-attachments/assets/1160ad4d-a236-44f2-8caa-27abaeb0137e" />
